.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "Dev Enviroment",
+
+	// Sets the run context to one level up instead of the .devcontainer folder.
+	"context": "..",
+
+	// Update the 'dockerFile' property if you aren't using the standard 'Dockerfile' filename.
+	"dockerFile": "../Dockerfile.vscode",
+
+	// Set *default* container specific settings.json values on container create.
+	"settings": { 
+		"terminal.integrated.shell.linux": null
+	},
+
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [],
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+
+	// Uncomment the next line to run commands after the container is created - for example installing curl.
+	// "postCreateCommand": "apt-get update && apt-get install -y curl",
+
+	// Uncomment when using a ptrace-based debugger like C++, Go, and Rust
+	// "runArgs": [ "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
+
+	// Uncomment to use the Docker CLI from inside the container. See https://aka.ms/vscode-remote/samples/docker-from-docker.
+	"mounts": [ "type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind"],
+
+	// Uncomment to connect as a non-root user if you've added one. See https://aka.ms/vscode-remote/containers/non-root.
+	// "remoteUser": "vscode"
+}
```

Comment:
Arquivos de configuração do editor/IDE, as vezes, salvam caminhos absolutos. Se esse for o caso e as configurações forem submetidas, elas provavelmente vão quebrar o editor/IDE de outro desenvolvedor que baixar o projeto. Eu sei que existe a revisão dos PR e tudo mais, porém é comum evitar subir essas configurações. Assim, é comum não submeter esses arquivos.

"E como podemos facilitar o desenvolvimento para quem quiser contribuir?"
- Criando um documento explicando como configurar o projeto usando vários tipos de editores e IDEs.
- Criando um script que você joga no terminal e ele configura o editor (não seguro).
- E por aí vai...

Reply:
Concordo, era para deixar como exemplo de uso. Já que não é ativo automaticamente.
Já removi. 👍🏽